operator/controllers/service_controller.go

```diff
@@ -18,13 +18,21 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"time"
 
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	lifecyclev1alpha1 "github.com/keptn-sandbox/lifecycle-controller/operator/api/v1alpha1"
+	"github.com/google/uuid"
+	"github.com/keptn-sandbox/lifecycle-controller/operator/api/v1alpha1"
```

Comment:
```suggestion
	keptnlfcv1alpha1 "github.com/keptn-sandbox/lifecycle-controller/operator/api/v1alpha1"
```
Please don't use the version only.